tutorials/beginner/introduction.ipynb

```diff
@@ -13,9 +13,9 @@
    "source": [
```

Comment:
<p>Could you try re-running this cell so we don't have the warning message when we push?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I think the comment is vague. If you can't identify which cell to re-run, you can really just re-run all. I think it was the first cell with a Jupyter error message